### PR TITLE
added conf-ppl package

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/conf-ppl/conf-ppl.1/opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+author: "R. Bagnara, P. M. Hill, E. Zaffanella, A. Bagnara et. al."
+maintainer: "francois.berenger@inria.fr"
+homepage: "http://bugseng.com/products/ppl/"
+license: "GPL"
+build: [
+  ["sh" "-c" "cc test.c -lppl_c -lppl -lgmpxx -lgmp"]
+]
+depends: [
+  "conf-gmp"
+  "conf-mpfr"
```
Comment (yallop) [inline packages/conf-ppl/conf-ppl.1/opam]: Can these dependencies be removed?  If gmp and mpfr are dependencies of ppl then it seems reasonable to leave the task of installing them to whatever/whoever installs ppl, ideally the system package manager.

File: packages/conf-ppl/conf-ppl.1/opam [hunk 2]
```diff
@@ -0,0 +1,17 @@
+opam-version: "1"
+author: "R. Bagnara, P. M. Hill, E. Zaffanella, A. Bagnara et. al."
+maintainer: "francois.berenger@inria.fr"
+homepage: "http://bugseng.com/products/ppl/"
+license: "GPL"
+build: [
+  ["sh" "-c" "cc test.c -lppl_c -lppl -lgmpxx -lgmp"]
```
Comment (yallop) [inline packages/conf-ppl/conf-ppl.1/opam]: Some of the link flags (e.g. `-lgmpxx` and `-gmp` don't appear to be required to build the test program.

**Comment by UnixJunkie** [comment]
Yes, you are right, only the ppl is a dependency in fact. I correct those.

**Comment by yallop** [comment]
Thanks!

**Pull request merged**